src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,26 @@
-export const Tabs = () => {};
+import { TabsColumn } from './TabsColumn/TabsColumn';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <TabsColumn
+              tab={tab}
+              activeTab={activeTab}
+              onTabSelected={tabId => onTabSelected(tabId)}
```

Comment:
```suggestion
              onTabSelected={onTabSelected}
```
You can pass this function dirrectly